db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/sqlgrammar.jj

```diff
@@ -3004,6 +3004,7 @@ TOKEN [IGNORE_CASE] :
 |	<PARQUET: "parquet">
 |   <AVRO: "avro">
 |	<ORC: "orc">
+|   <OPTIMIZE: "optimize">
```

Comment:
Is there a need to make this a reserved keyword instead of a non-reserved keyword?  If by any chance a customer is using optimize as a column name or table name this change would break use of that object.

Reply:
Set OPTIMIZE to the non-reserved keyword. Thanks @msirek pointed out the potential upgrade issue.